code/billy/lab_08.py

```diff
@@ -0,0 +1,233 @@
+'''
+program: lab_08
+author: billy frick
+date: 28 september 2022
+
+function:   > this program will pull a 'contacts list' from a CSV file.
+            > the program will open the file and create a dictionary based on the information given.
+            > the user can then update, add or delete a contact; as well as print a specific contact and their related information.
+            > when the user inputs 'exit' as a command, the program will exit and the contact CSV will be updated by writing (w) to it.
+'''
+
+# this function will open the csv file and split the headers and contacts.
+# the contacts will be seperated by using '\n', and the information of each contact will be seperated using ','.
+def open_csv(file):
```

Comment:
Great comments on what the code does!

* One thing you can try out is using [Docstrings](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/docs/Docstrings.md)
    * This is a great way to document what the code is doing and this docstring will show up in many automated activities. Like when you hover over the part of code where the function is called, you will get a popup that shows the doctstring as well as options to go to function definition.